### PR TITLE
Add new blog post: How our OSPO uses Batch Changes...

### PR DESCRIPTION
**Pull request by jdorfman**
### Changelog

- Added a new blog post
- Added Scarf pixel, I don't trust GA

### Test

1. Ensure prettier has standardized the proposed changes.

### CODE REVIEW

File: content/blogposts/2022/batch-changes-ospo.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: 'How our OSPO uses Batch Changes to automatically add license files'
```
Comment (slimsag) [inline content/blogposts/2022/batch-changes-ospo.md:2]: I kept hearing OSPO and had no idea what it was. I imagine >90% of people who read this title will also have no clue what it is.

Added to my infamous [WDYMWDIM](https://docs.google.com/spreadsheets/d/1E71lKEMl6sFvey0BAVWyQih2an64bSutZfBJ6haXY9c/edit#gid=0) sheet.

Comment (slimsag) [reply content/blogposts/2022/batch-changes-ospo.md:2]: Alternative title:

> How we analyzed hundreds of repositories to ensure they had open source licenses

Comment (jdorfman) [reply content/blogposts/2022/batch-changes-ospo.md:2]: If OSPO is too niche then this post will be missing a large audience.

> How we analyzed hundreds of repositories to ensure they had open source licenses

I love it.

**Review by mplacona — APPROVED** [review]
🚢

**Pull request merged**